### ISSUE TITLE
Vertical whitespace between paragraphs is lost when message includes quote

### ISSUE THREAD

**Issue by Johennes** [vector-im/element-web#24419]
### Steps to reproduce

```
> aaa

aaa

aaa

aaa
```

is rendered as

![quote](https://user-images.githubusercontent.com/1137962/216427837-456bea9c-7384-415d-a3a9-576439c98400.png)

whereas

```
aaa

aaa

aaa
```

is rendered as

![Screenshot 2023-02-02 at 20 15 00](https://user-images.githubusercontent.com/1137962/216427977-4e1d89f3-f8ea-4e29-9f9d-169f6774527c.png)

#### What did you expect?

In both cases there should be vertical whitespace between the paragraphs.

#### What happened instead?

When a quote is included the vertical whitespace is collapsed making the text look like a single paragraph.

### Operating system

macOS

### Application version

Element Desktop nightly

### Will you send logs?

No

**Comment by daniellekirkwood** [comment]
Not sure of reproducing but here's an example: 

<img width="245" alt="Screenshot 2023-02-02 at 20 05 17" src="https://user-images.githubusercontent.com/89144281/216438023-675fe17e-52b1-4f50-be1d-85b429331534.png">
<img width="233" alt="Screenshot 2023-02-02 at 20 05 22" src="https://user-images.githubusercontent.com/89144281/216438025-f8fe3a47-24c4-4049-b5c5-9c799e03e664.png">

**Comment by Johennes** [comment]
@alunturner looks like this was caused by https://github.com/matrix-org/matrix-react-sdk/commit/39f5fc034f82de7843c0570df5fa65226ac5c677. Could you give this a look please?

**Comment by weeman1337** [comment]
Paragraphs are lost if the message contains any markdown.

**Comment by alunturner** [comment]
Addressed in [PR 10071 in the react sdk](https://github.com/matrix-org/matrix-react-sdk/pull/10071)
